.github/ISSUE_TEMPLATE/kubernetes_update.md

```diff
@@ -38,10 +38,12 @@ List of items to do for upgrading to {version_1} {version_2} {version_3}
         - [ ] `dev_eks_example_cluster_version`, which is the newest EKS supported version in {version_1} {version_2} {version_3}
         - [ ] `dev_minikube_example_cluster_version`, which is {version_2} with the supported patch version
 - [ ] If client-go pulled in a new version of gRPC, then also
-    - [ ] Update the SDK [base image grpc version](https://github.com/googleforgames/agones/blob/main/build/includes/sdk.mk#L30) and rebuild the image. Note that this can take a while and in the past we have had to manually push it to gcr because cloud build doesn't like how long it takes.
-    - [ ] Regenerate allocated API endpoints: [make gen-allocation-grpc](https://github.com/googleforgames/agones/blob/main/build/includes/allocation.mk#L55)
-    - [ ] Regenerate all client sdks: [make gen-all-sdk-grpc](https://github.com/googleforgames/agones/blob/main/build/README.md#make-gen-all-sdk-grpc)
-    - [ ] Update the version number in C++ Cmake scripts [here](https://github.com/googleforgames/agones/blob/main/sdks/cpp/CMakeLists.txt#L100) and [here](https://github.com/googleforgames/agones/blob/main/sdks/cpp/cmake/prerequisites.cmake#L34)
+    - [ ] Update the `grpc_release_tag` in the SDK [base image grpc version](https://github.com/googleforgames/agones/blob/main/build/includes/sdk.mk).
+    - [ ] Update the gRPC version number in C++ CMake scripts, `AGONES_GRPC_VERSION` [here](https://github.com/googleforgames/agones/blob/main/sdks/cpp/CMakeLists.txt)
+          and `gRPC_GIT_TAG` [here](https://github.com/googleforgames/agones/blob/main/sdks/cpp/cmake/prerequisites.cmake)
+    - [ ] Regenerate all client sdks: [make gen-all-sdk-grpc](https://github.com/googleforgames/agones/blob/main/build/README.md#make-gen-all-sdk-grpc) 
+          This can take an hour or so, as the above changes force a rebuild. Plan your day accordingly 😃.
+    - [ ] Regenerate allocated API endpoints: [make gen-allocation-grpc](https://github.com/googleforgames/agones/blob/main/build/README.md#make-gen-allocation-grpc)
```

Comment:
First, we need to run `make gen-allocation-grpc`, followed by the make gen-all-sdk-grpc target🤔
Do we need also mention to run `make build` target followed by the sdk.mk file update and rebuilding all the sdk docker images if it is required?

Reply:
> First, we need to run `make gen-allocation-grpc`, followed by the make gen-all-sdk-grpc target🤔 

No I deliberately switched the order to make `make gen-all-sdk-grpc` run first. It's the more important step of the two.


> Do we need also mention to run `make build` target followed by the sdk.mk file update and rebuilding all the sdk docker images if it is required?

There's no need to run `make build`, everything that needs to get rebuilt gets rebuilt automatically (the sdk and allocation grpc build images are actually separate from the main Go/Agones build image).

Reply:
I am super confused, but if these steps work for the Kubernetes upgrade, then it's fine.😅